pypeit/flatfield.py

```diff
@@ -1648,12 +1655,11 @@ def spectral_illumination(self, gpm=None, debug=False):
             gpm = self.rawflatimg.select_flag(flag='BPM', invert=True)
 
         # Obtain relative spectral illumination
+        spat_flexure = self.wavetilts.spat_flexure if self.wavetilts is not None else np.zeros((self.slits.nslits, 2))
```

Comment:
Do we need to define this as an array of 0s when `wavetilts` is None, or can we set `spat_flexure = None`?